docs/operations/metrics.md

```diff
@@ -298,6 +298,9 @@ If the JVM does not support CPU time measurement for the current thread, `ingest
 |`worker/task/assigned/count`|Number of tasks assigned to an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
 |`worker/task/completed/count`|Number of tasks completed by an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
 |`worker/task/running/count`|Number of tasks running on an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
+|`s3upload/threadPool/taskQueuedDuration`|Milliseconds spent by a task in queue before it starts uploading a part to S3 (in a multi-part upload) when durable storage is enabled.||Varies|
+|`s3upload/threadPool/queuedTasks`|The number of tasks that are currently queued and waiting to upload a part to S3 (in a multi-part upload) when durable storage is enabled.||Varies|
+|`s3upload/threadPool/taskDuration`|The time taken in milliseconds to upload a part to S3 (in a multi-part upload) when durable storage is enabled.|`uploadId`, `partNumber`|Varies|
```

Comment:
Better names:

```
s3upload/chunk/uploadTime
s3upload/chunk/queuedTime
s3upload/chunk/queueSize
```

Please also emit a metric from `RetryableS3OutputStream` which tracks total time taken to upload all chunks of a job, say (`s3upload/job/totalTime`). That would give us a good measure of the parallelism:

parallelism is good if

```
s3upload/job/totalTime < sum(s3upload/chunk/uploadTimeMillis)
```

Reply:
@kfaraz 

Have renamed the metrics.

Also have added metrics for `s3upload/job/totalTime` and `s3upload/job/totalBytes`. (Added the latter one as there was an older todo comment in the code for `totalChunkSize` saying `This should be emitted as a metric`

Hope that works!